src/data_info/routes/groups.clj

```diff
@@ -0,0 +1,63 @@
+(ns data-info.routes.groups
+  (:use [common-swagger-api.schema]
+        [otel.middleware :only [otel-middleware]]
+        [data-info.routes.schemas.users])
+  (:require [data-info.services.groups :as groups]
+            [schema.core :as s]
+            [ring.util.http-response :refer [ok]]))
+
+(s/defschema Group
+  {:group-name NonBlankString
+   :members [NonBlankString]})
+
+(s/defschema GroupMembers
+  (dissoc Group :group-name))
```

Comment:
I wasn't sure if these endpoints were even going to be added to terrain (which I guess is also true of the field naming thing, but I should just be less verbose there anyway), since they're being added in order for a backend service to be able to update the members of groups (the UI will presumably continue interfacing directly with grouper, with the intermediate services sending AMQP or NATS messages to trigger updates of the groups in irods). I guess it doesn't hurt to add them to `common-swagger-api` regardless, so maybe I should just go ahead and do that? Not sure.

Reply:
Oh right. I didn't think of that. If these endpoints aren't going into terrain for public-facing clients (like our UI), then I wouldn't bother with `common-swagger-api` until we actually need it (otherwise it's an extra unnecessary place to update these docs in the future).

Reply:
Yeah. These are mostly admin-only type endpoints anyway (only listing group members isn't), so I'm not sure there's much use for bubbling this stuff up into the API or UI.